travis-ci.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set +e
+
+if [ -z "$TRAVIS" ]; then
+    echo "this script is intended to be run only on travis" >&2
+    exit 1
+fi
+
+KUBECTL_WAIT_CMD="kubectl wait --timeout=5m --for=condition=Ready pod/sonobuoy -n heptio-sonobuoy"
+SONOBUOY_CMD=./sonobuoy
+SONOBUOY_SUB_CMD=run -m Quick
+
+function waitForDeployment() {
```

Comment:
I have a PR nearly ready (maybe putting it up tomorrow) which would introduce the `--wait` options to run and delete commands. This makes scripting this much easier.